examples/teams/team-burnham/index.ts

```diff
@@ -12,10 +12,11 @@ function getUserArns(scope: Construct, key: string): ArnPrincipal[] {
 }
 
 export class TeamBurnham extends ApplicationTeam {
-    constructor(scope: Construct) {
+    constructor(scope: Construct, policyDir: string) {
         super({
             name: "burnham",
-            users: getUserArns(scope, "team-burnham.users")
+            users: getUserArns(scope, "team-burnham.users"),
+            networkPoliciesDir: policyDir
```

Comment:
`policyDir` v `networkPoliciesDir`...lets use a consistent name.

Reply:
rewritten to networkPoliciesDir